open_academy/views/course_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="course_view_form"  model="ir.ui.view">
+        <field name="name">course.form</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <form string="Form">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name"/>
+                        </group>
+                        <notebook>
+                            <page string="Description">
+                                <field name="description"/>
+                            </page>
+                        </notebook>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="course_view_search"  model="ir.ui.view" >
+        <field name="name">course.view.search</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="title"/>
+                <field name="description"/>
+            </search>
+        </field>
+    </record>
+    <record id="courses_action" model="ir.actions.act_window">
+        <field name="name">Courses</field>
+        <field name="res_model">course</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem id="course_menu_root" name="Open Academy">
+        <menuitem id="course_menu"  name="Courses"  action="courses_action" /> 
+        <menuitem id="session_menu"   name="Sessions"  action="sessions_action" />
```

Comment:
Add a new record for this session menu, move the record to session_views

Reply:
each view will have his own record with his own menu? or  I just need one which is gonna be inside sesssion_views?

Reply:
I think that I already have one record for the session_menu on session_views

Reply:
`<record id="session_action" model="ir.actions.act_window" >
        <field name="name">Sessions</field>
        <field name="res_model">session</field>
        <field name="view_mode">tree,form</field>
    </record>
`

Reply:
what i try to said is that remove the menu item session from this file and create a new one in session file for session like:
```
<menuitem id="session_menu_root" name="Open Academy">
        <menuitem id="session_menu"   name="Sessions"  action="sessions_action" />
    </menuitem>
```